composer.json

```diff
@@ -18,6 +18,7 @@
     ],
     "require": {
         "php": "^7.3|^8.0",
+        "sixlive/nova-text-copy-field": "^1.5",
```

Comment:
<img width="1079" alt="Screen Shot 2021-04-18 at 11 31 01 PM" src="https://user-images.githubusercontent.com/24723492/115181724-2458ac00-a09e-11eb-89eb-d7f2af8db79d.png">

Reply:
Can you make the url clickable and open a new tab? That copy icon is great.

Reply:
I will probably need to make a custom fork of the repo I imported to pull this off. Doesn't appear like it supports `asHtml` right now but I can probably add that in via my own forked version of it.

Reply:
That sounds good. We can add a forked version of it later to this package.